.gitignore

```
@@ -26,3 +26,4 @@ lerna-debug.log*
 .tuono
 out
 target
+.local
```

Why not using the same pattern you used in https://github.com/tuono-labs/tuono/pull/619?
```suggestion
.env*.local
```